### PR TITLE
some fixes on Windows

### PR DESCRIPTION
**Pull request by brucelane**
- renamed uint to unsigned int
- added include
- added sample

### CODE REVIEW

File: include/UnionJack.h [hunk 1]
```diff
@@ -9,11 +9,12 @@
 #ifndef __UnionJack__
 #define __UnionJack__
 
+#include "cinder/gl/gl.h"
 #include "cinder/gl/Vbo.h"
 
 class UnionJack {
 public:
-    UnionJack( uint length = 10 );
+	UnionJack(unsigned int length = 10);
```
Comment (drewish) [inline include/UnionJack.h:17]: Do you mind switching the tabs back to spaces? And putting the spaces back inside the parens?

**Comment by drewish** [comment]
is the `uint` -> `unsigned int` to get it compiling on windows?

**Comment by drewish** [comment]
What do you think about going to `uint32_t` instead?

**Comment by brucelane** [comment]
Yes with visual studio, uint32_t seems fine

**Comment by drewish** [comment]
Cool I'll switch that since some of my changes from last night caused conflicts with this

**Comment by drewish** [comment]
Do you want to switch this up to add the project and sample app?

**Comment by brucelane** [comment]
I fix this with your latest changes, then do a new PR. You can close this

**Pull request closed without merging**